### DISCUSSION TITLE
How to resize and position new window in WPF C#?

### DISCUSSION THREAD

**Discussion by Dave-Jalpan** [MicrosoftEdge/WebView2Feedback#3882] [Q&A]
I want to resize and position newly open window 
```
private void CoreWebView2_NewWindowRequested(object sender, Microsoft.Web.WebView2.Core.CoreWebView2NewWindowRequestedEventArgs e)
{
    /// How to resize and position newly open window in WPF????? 
}

```

**Comment by Symbai** [comment]
Read the [docs](https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2newwindowrequestedeventargs?view=webview2-dotnet-1.0.864.35#properties) especially the part of `WindowFeatures`
